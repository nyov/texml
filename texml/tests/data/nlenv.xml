<TeXML>
	<!-- Default is nlX=1 -->
	x <env name="y">?</env> z
	<!-- Explicit nlX=1 -->
	x <env name="y" nl1="1" nl2="1" nl3="1" nl4="1">?</env> z
	<!-- No newlines at all -->
	x <env name="y" nl1="0" nl2="0" nl3="0" nl4="0">?</env> z
	<!-- Check each attribute -->
	x <env name="y" nl1="1" nl2="0" nl3="0" nl4="0">?</env> z
	x <env name="y" nl1="0" nl2="1" nl3="0" nl4="0">?</env> z
	x <env name="y" nl1="0" nl2="0" nl3="1" nl4="0">?</env> z
	x <env name="y" nl1="0" nl2="0" nl3="0" nl4="1">?</env> z
</TeXML>
