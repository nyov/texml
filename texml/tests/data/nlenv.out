x
\begin{y}
?
\end{y}
z x
\begin{y}
?
\end{y}
z x \begin{y}?\end{y} z x
\begin{y}?\end{y} z
	x \begin{y}
?\end{y} z
	x \begin{y}?
\end{y} z
	x \begin{y}?\end{y}
z
